content/en/docs/components/pipelines/quickstart.md

```diff
@@ -3,4 +3,340 @@ title = "Quickstart"
+```
+**Warning:** Using `SCOPES="cloud-platform"` grants all GCP permissions to the cluster. For a more secure cluster setup, refer to [Authenticating Pipelines to GCP][authenticating-pipelines-gcp].
+
+### 2) Deploy a KFP standalone instance into your cluster
+Now that you have a Kubernetes cluster, you can deploy a KFP standalone instance into this cluster. Replace `PIPELINE_VERSION` with the desired version of KFP and run the following script:
+
+```shell
+export PIPELINE_VERSION="2.0.0-alpha.3"
+
+kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION"
+kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
+kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION"
+```
+
+If you've deployed KFP on a local cluster, run the following to port forward the KFP Dashboard:
+
+```shell
+kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
+```
+
+then visit [http://localhost:8080][localhost] to view the KFP Dashboard.
+
+If you've deployed KFP on a cluster on GCP, run the following to get the public URL for the KFP Dashboard:
+
+```shell
+kubectl describe configmap inverse-proxy-config -n kubeflow | grep googleusercontent.com
+```
+
+Open the returned URL in your browser to view the KFP Dashboard.
+
+### 3) Compose and submit simple pipeline
+Now we'll compose a pipeline and submit it for execution by KFP using the KFP SDK.
+
+To install the KFP SDK (v2 is currently in pre-release), run:
+```shell
+pip install kfp --pre
+```
+
+The following simple pipeline adds two integers together then adds another integer to this result to come up with a final sum.
+
+```python
+from kfp import dsl
+from kfp import client
+
+
+@dsl.component
+def addition_component(num1: int, num2: int) -> int:
+    return num1 + num2
+
+
+@dsl.pipeline(name='addition-pipeline')
+def my_pipeline(a: int, b: int, c: int = 10):
+    add_task_1 = addition_component(num1=a, num2=b)
+    add_task_2 = addition_component(num1=add_task_1.output, num2=c)
+
+
+endpoint = '<KFP_ENDPOINT>'
+kfp_client = client.Client(host=endpoint)
+run = kfp_client.create_run_from_pipeline_func(
+    my_pipeline,
+    arguments={
+        'a': 1,
+        'b': 2
+    },
+)
+url = f'{endpoint}/#/runs/details/{run.run_id}'
+print(url)
+```
+
+Let's walk through each part step-by-step.
+
+<!-- TODO: add lightweight component authoring link -->
+First, we begin by creating a lightweight Python component using the `@dsl.component` decorator:
+```python
+@dsl.component
+def addition_component(num1: int, num2: int) -> int:
+    return num1 + num2
+```
+This decorator transforms a Python function into a component that can be used within a pipeline. Type annotations on the parameters and return value are required, as this tells the KFP executor how to serialize and deserialize data as it is passed between components. It also allows the KFP compiler to type check your pipeline.
```

Comment:
> type check your pipeline

type check data-passing between components?

Reply:
Updated to something similar